persistence/src/commonMain/db_global/com/wire/kalium/persistence/Accounts.sq

```diff
@@ -0,0 +1,51 @@
+import com.wire.kalium.persistence.dao.QualifiedIDEntity;
+import com.wire.kalium.persistence.model.LogoutReason;
+
+CREATE TABLE Accounts (
```

Comment:
AFAIK we usually use snake_case not camelCase for databases column naming. I think we should follow it here as well.